src/constants.ts

```diff
@@ -30,6 +30,7 @@ export const DEFAULT_SETTINGS: Omit<ObsidianGitSettings, "autoCommitMessage"> =
         refreshSourceControl: Platform.isDesktopApp,
         basePath: "",
         differentIntervalCommitAndPush: false,
+        globalEnable: true,
```

Comment:
I would rename this to `temporaryDisableAutomatics`
So the toggle gets a new meaning as well. I think it's better to add a toggle to just disable all automatics all together. So push and pull as well.